src/language-js/index.js

```diff
@@ -160,7 +160,7 @@ const languages = [
     extensions: [".json5"],
     filenames: [".babelrc"],
     linguistLanguageId: 175,
-    vscodeLanguageIds: ["jsonc"]
+    vscodeLanguageIds: []
```

Comment:
Are there any language IDs for JSON5?

Reply:
There's no official language id for JSON5, but there's a unofficial one (`json5`) from [community extension](https://github.com/mrmlnc/vscode-json5).

Should we use that instead?

Reply:
Sounds good. Providing an unofficial one should be fine if there isn’t an official one.